### ISSUE TITLE
Option to reset chain state if shortcut released

### ISSUE THREAD

**Issue by kdorsel** [jigish/slate#254]
Given a sample config :
...
var leftMove = S.op("chain", {"operations":[leftHalf, leftThird, leftQuarter]});
S.bnda({"left:alt,ctrl,cmd" : leftMove});
...

Let's assume I activate the shortcut twice and get to leftThird. Later I want to make it fill the leftHalf, although currently if I activate the shortcut it fills leftQuarter.

I would like to see either a global or per bind option that resets the chain state once the shortcut is released.

**Comment by contentfree** [comment]
Yeah, having something like `config secondsBeforeRepeat` that resets the chain would be very useful.
